admin/json.go

```diff
@@ -21,6 +21,11 @@ func writeJSONError(w http.ResponseWriter, err *HttpError) {
 	writeJSON(w, response)
 }
 
+func writeJSONOk(w http.ResponseWriter) {
+	response := make(map[string]string)
```

Comment:
Seems like you can do this by declaring an empty map and initializing it in an init() function:

```
var emptyMap *map[string]string
func init() {
  if emptyMap == nil {
    m = make(map[string]string)
    emptyMap = &m
  }
}
```

`init()` functions are threadsafe.